waffle_utils/video_processing/video_tools.py

```diff
@@ -0,0 +1,177 @@
+import os
+from pathlib import Path
+
+import cv2
+
+from waffle_utils.utils import type_validator
+
+
+class VideoTools:
+    def __init__(self, input_path: str, output_path: str):
+        self.input_path = self._input_path = input_path
```

Comment:
It seems `self.input_path` can get `image_paths` and `video_paths`.
I think this is not safe.

And is `VideoTools` is right place to `create_video` function?